src/hooks/v2v3/transactor/TransferUnclaimedTokensTx.ts

```diff
@@ -17,12 +21,26 @@ export function useTransferUnclaimedTokensTx(): TransactorInstance<{
   const { transactor } = useContext(TransactionContext)
   const { contracts } = useContext(V2V3ContractsContext)
   const { tokenSymbol } = useContext(V2V3ProjectContext)
-  const { projectId } = useContext(ProjectMetadataContext)
+  const { projectId, cv } = useContext(ProjectMetadataContext)
 
   const { userAddress } = useWallet()
 
   return ({ amount, to }, txOpts) => {
-    if (!transactor || !projectId || !contracts?.JBTokenStore) {
+    try {
```

Comment:
**Fresh**

Reply:
I've applied the same behavior for `v1` version of this function in the latest commit.